extensions/KHR/SPV_KHR_bfloat16.asciidoc

```diff
@@ -0,0 +1,201 @@
+SPV_KHR_bfloat16
+================
+
+Name Strings
+------------
+
+SPV_KHR_bfloat16
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Registry
+
+Contributors
+------------
+
+- Tobias Hector, AMD
+- Stu Smith, AMD
+- Jeff Bolz, Nvidia
+- Kévin Petit, Arm
+- David Neto, Google
+- Graeme Leese, Broadcom
+- Ruihao Zhang, Qualcomm
+- Mark Sheppard, Imagination
+- Ben Ashbaugh, Intel
+- Dmitry Sidorov, Intel
+- Victor Mustya, Intel
+
+Notice
+------
+
+Copyright (c) 2023-2025 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- Complete
```

Comment:
Can you please update the extension status to include the ratification approval dates?

```suggestion
- Approved by the SPIR-V Working Group: 2025-01-29
- Approved by the Khronos Board of Promoters: 2025-03-14
```